src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -732,6 +732,17 @@ public ConsoleColorProxy(ConsoleHostUserInterface ui)
                 _ui = ui;
```

Comment:
@SteveL-MSFT, your last commit had 3 failures in `PowerShell-CI-windows`
Tests for $ErrorView.ConciseView tests.Script error should contain path to script and line for error
```powershell
Invalid pattern 'C:\Users\VssAdministrator\AppData\Local\Temp\10a1b70a-ecdc-42d3-87ec-82f2eee3a55f\test.ps1' at offset 4. Unrecognized escape sequence \\U.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Formatting\ErrorView.Tests.ps1: line 39
39:             $e | Out-String | Should -Match $testScriptPath

```
Import-PSSession Cmdlet error handling.Test content and format of proxied error message (Windows 7: #319080).Test non-terminating error
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Implicit.Remoting.Tests.ps1: line 209
209:                 ($errorString -like "*VariableNotFound*") | Should -BeTrue
```
Write-Error Tests.ErrorRecord should not be truncated or have inserted newlines when redirected from another process
```powershell
Expected exactly 4, but got 1.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Write-Error.Tests.ps1: line 106
106:         $result.Count | Should -BeExactly 4
```